File: BookMyshowFrontend/app/src/app/shared/models/movie.ts
export class Movie {
    id: number;
    movieName: string;
    languages: string;
    hours: string;
    genre: string;
    dateOfRelease: string;
    locationNames: string;
    constructor(id: number, movieName: string, languages: string, hours: string, genre: string, dateOfRelease: string, locationNames: string,
    ) {
        this.id = id;
        this.movieName = movieName;
        this.languages = languages;
        this.hours = hours;
        this.genre = genre;
        this.dateOfRelease = dateOfRelease;
        this.locationNames = locationNames;
    }
}

File: BookMyShowAPI/Data/DataModels/Movie.cs
﻿using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Data.DataModels
{
    [Table("Movie")]
    public class Movie
    {
        [Column("Id")]
        [Key]
        public int Id { get; set; }
        [Column("MovieName")]
        public string? MovieName { get; set; }
        [Column("Language")]
        public string Language { get; set; }
        [Column("Hours")]
        public string Hours { get; set; }
        [Column("Genre")]
        public string Genre { get; set; }
        [Column("DateOfRelease")]
        public DateTime DateOfRelease { get; set; }
    }
}
